elliottlib/bzutil.py

```diff
@@ -82,6 +82,33 @@ def get_bugs_map(self, ids: List[int], raise_on_error: bool = True, **kwargs) ->
             id_bug_map[ids[i]] = bug
         return id_bug_map
 
+    def create_bug(self, bugtitle, bugdescription, keywords: List):
+        createinfo = self._client.build_createbug(
+            product=self.config.get('product'),
+            version=self.config.get('version')[0],
+            component="Release",
+            summary=bugtitle,
+            keywords=keywords,
+            description=bugdescription)
+        newbug = self._client.createbug(createinfo)
+        # change state to VERIFIED, set target release
+        try:
+            update = self._client.build_update(status="VERIFIED", target_release=self.config.get('target_release'))
```

Comment:
Can you accept a state, that way this method would be more general. And then pass VERIFIED from placeholder/textonly methods

Reply:
updated